docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -225,6 +228,18 @@ property definition to the ``_meta.presto`` property of the target index mapping
         }
     }'
 
+Date types
+^^^^^^^^^^
+
+Elasticsearch supports a wide array of `date`_ formats including `built-in date formats`_ and also `custom date formats`_.
+Currently, the elasticsearch connector supports only default **date** type. All other date formats including `built-in date formats`_ and `custom date formats`_ are not supported.
```

Comment:
The Elasticsearch connector supports only ..

Reply:
Addressed.

Reply:
Doesnt look like it